<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1617</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resource/Icons/NovPacient.png</normaloff>Resource/Icons/NovPacient.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BTNKontrolneMeritve">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vnos podatkov kontrolnih meritev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTNDataloger">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vnos podatkov Datalogerja</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1601</width>
      <height>881</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTNIzhod">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Izhod</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="DateEditZacetek">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="DateEditKonec">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="RadioButtonH">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Vodostaj</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="RadioButtonT">
       <property name="text">
        <string>Temperatura</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1617</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatoteka">
    <property name="title">
     <string>Datoteka</string>
    </property>
    <addaction name="exitAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoƒç</string>
    </property>
    <addaction name="actionNavodilo_za_uporabo"/>
    <addaction name="separator"/>
    <addaction name="ActionOprogramu"/>
   </widget>
   <addaction name="menuDatoteka"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShrani_novega_pacienta">
   <property name="text">
    <string>Shrani pacienta</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Izhod</string>
   </property>
  </action>
  <action name="ActionOprogramu">
   <property name="text">
    <string>O programu</string>
   </property>
  </action>
  <action name="actionNavodilo_za_uporabo">
   <property name="text">
    <string>Navodilo za uporabo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
